pkg/views/gitprovider/select.go

```diff
@@ -125,6 +145,36 @@ func GitProviderSelectionView(gitProviderAddView *apiclient.SetGitProviderConfig
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	tokenDataForm := huh.NewForm(
+		huh.NewGroup(
+			huh.NewInput().
+				Title("Enter the organization name associated with this token").
```

Comment:
@the-johnwick please leave comments under mine instead of in the root thread.

> @Tpuljak https://github.com/daytonaio/daytona/pull/1032#discussion_r1754413691 I agree but the problem is that we don't have to fetch the namespaces and repos using the token that the user wants to add.

You will need to create new endpoints for this that can take in a token and `ProviderId` and return namespaces and repos.